SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -661,7 +697,8 @@ void CMeshSolver::SetBoundaryDisplacements(CGeometry *geometry, CNumerics *numer
     if ((config->GetMarker_All_Deform_Mesh(iMarker) == YES) ||
         (config->GetMarker_All_Moving(iMarker) == YES)) {
 
-      BC_Deforming(geometry, numerics, config, iMarker);
+      if (!velocity_transfer) BC_Deforming(geometry, numerics, config, iMarker);
```

Comment:
Should this boolean be used to prevent the section with GetSurface_Movement(DEFORMING) from being applied when there is velocity?

Reply:
Agreed. I've added the velocity boolean to prevent this since currently there is no support for dynamic FSI problems with additional enforced surface motion.